README.md

```diff
@@ -233,25 +195,120 @@ print status
   - `backoff_factor` - a backoff factor to apply between retry attempts.
 
 
+### Friendbot
```

Comment:
emphasize that it is a service for creating accounts, and that it's not relevant and exists on production